packages/components/src/badge/stories/index.tsx

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { Card, CardBody } from '@wordpress/components';
+import { Story } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { Badge, BadgeProps } from '../';
+
+const Template: Story< BadgeProps > = ( args ) => (
+	<Card>
+		<CardBody>
+			<Badge { ...args } />
+		</CardBody>
+	</Card>
+);
+
+export const Primary = Template.bind( {} );
+
+Primary.args = {
+	count: 15,
```

Comment:
This and the template satisfy storybooks args feature letting you interactively change the args passed to the component. A good follow up PR will be to convert more of the stories to support this feature.